README.md

```diff
@@ -1 +1,29 @@
-Please see the project homepage https://github.com/CiscoCTA/taxii-log-adapter/wiki
\ No newline at end of file
+Please see the project homepage https://github.com/CiscoCTA/taxii-log-adapter/wiki
+
+#### RELEASE NOTES
+
+##### Version 2.0.0
+
+* Due to migration to Spring Boot 2.0 we renamed configuration property `taxiiService` to `taxii-service`
+  because camel case configuration properties are no longer supported.
+
+* Slash character `/` in JSON output is now escaped by default and is returned as `\/` according to changes in Saxon-HE 9.7.0-8.
+  When you'd like to keep backward compatibility you need to modify stylesheet and use e.g. character maps as shown in
+  following example:
```

Comment:
Slash character `/` in JSON output is now escaped by default and it's returned as `\/` according to the changes in Saxon-HE 9.7.0-8.
If you'd like to keep backward compatibility you can modify the stylesheet and use character maps as shown in the  following example: